metroae

```diff
@@ -1085,6 +1089,82 @@ function check_if_mount_point_configured {
     fi
 }
 
+function update_metroae {
+    debug ${FUNCNAME[0]}
+    declare -l confirmation
+    if [[ $RUN_MODE == "CONTAINER" ]]; then
+        if [[ -z $2 ]]; then
+            confirmation="init"
+            while [[ $confirmation != "n" ]] && [[ $confirmation  != "y" ]] && [[ $confirmation != "" ]]
+            do
+                echo ""
+                read -p "Do you really want to update the MetroAE Script? (y/N): " confirmation
```

Comment:
I would prefer

echo ""
echo "We recommend that you update your metroae command to the same version as the container you are loading."
read -p "Would you like us to update the metroae script? (Y/n): " confirmation

Notice that I changed the default to "Y/n". I want "enter" to mean Y. This requires changes, below.